main.tf

```diff
@@ -0,0 +1,173 @@
+data "aws_iam_policy_document" "assume_role" {
+  statement {
+    effect = "Allow"
+
+    principals {
+      type        = "Service"
+      identifiers = ["lambda.amazonaws.com"]
+    }
+
+    actions = ["sts:AssumeRole"]
+  }
+}
+
+data "aws_iam_policy_document" "lambda_logging" {
+  statement {
+    effect = "Allow"
+
+    actions = [
+      "logs:CreateLogGroup",
+      "logs:CreateLogStream",
+      "logs:PutLogEvents",
+    ]
+
+    resources = ["arn:aws:logs:*:*:*"]
```

Comment:
Though this doesn't have much impact to the security, try to follow the principle of least privilege
narrow down the scope of resource, something like arn:aws:logs:ca-central-1:_accountId_:/aws/lambda/_lambdaFunctionName_*